### ISSUE TITLE
RuntimeError: can't access GPIO on Raspberry PI

### ISSUE THREAD

**Issue by sun-jkp** [nRF24/pyRF24#58]
I have ran a example code from [http://pyrf24.rtfd.io](https://pyrf24.readthedocs.io/en/latest/examples.html)
getting_started example.
I got this error.
```
/home/rpi/doornRf24/getstart.py
Which radio is this? Enter '0' or '1'. Defaults to '0' 0
Traceback (most recent call last):
  File "/home/rpi/doornRf24/getstart.py", line 48, in <module>
    if not radio.begin():
           ^^^^^^^^^^^^^
RuntimeError: can't access /sys/class/gpio/gpio%d/direction GPIO pin. check access rights
```
Environment:
- Device: raspberry pi 4
- OS: Bookworm

I would like to know how to fix it.
Thanks!

**Comment by 2bndy5** [comment]
Try [installing from source](https://pyrf24.readthedocs.io/en/latest/#installing-from-github) instead of installing from pypi.

Are you using a RPi5? The GPIO interaction on Linux has changed since RPi5 came out. We made source changes that should be backward compatible, but you'll need to restart your RPi after installing the newer source changes (which hasn't been released yet).

**Comment by 2bndy5** [comment]
I just published v0.3.0, but it will be a few hours until it gets uploaded to pypi.

**Comment by sun-jkp** [comment]
> I just published [v0.3.0](https://pypi.org/project/pyrf24/0.3.0/) to pypi, but it might be a day or two until the [piwheels project compiles the 32-bit binary distributions](https://www.piwheels.org/project/pyrf24/) (only used for RPi OS 32-bit).

After, I update to v0.3.0.
It work!
Thank you!
